frontend/src/screens/wallet/Receive.tsx

```diff
@@ -116,6 +127,28 @@ export default function Receive() {
         title="Receive"
         description="Create a lightning invoice that can be paid by any bitcoin lightning wallet"
       />
+      {!!channels?.length && (
```

Comment:
need to update this check to also only happen if the backend type has channel management (the useInfo hook exposes this, see other places in the source code)

Reply:
(also applies to send)